### PR TITLE
Will/orc 259 updates to  app service and function app

### PR DESCRIPTION
**Pull request by WillBeebe**
I've tested all the underlying components. I'm going to use these to deploy two fresh apps to do another clean test.

Screenshot 2023-01-30 at 7 39 29 PM

### CODE REVIEW

File: azure-function-app/src/main.tf [hunk 1]
```diff
@@ -1,12 +1,25 @@
...
+  monitoring = {
+    mode = "AUTOMATED"
+  }
   dns                = var.dns
-  network            = var.network
-  monitoring         = var.monitoring
   virtual_network_id = var.azure_virtual_network.data.infrastructure.id
   location           = var.azure_virtual_network.specs.azure.region
   contact_email      = var.md_metadata.target.contact_email
+  md_metadata        = var.md_metadata
```
Comment (WillBeebe) [inline azure-function-app/src/main.tf:24]: The alarm channel needs it. We need to back out `md_metadata` from there first.

File: azure-app-service/massdriver.yaml [hunk 2]
```diff
@@ -144,29 +143,13 @@ params:
...
           type: string
           oneOf:
-            - title: 1 vCPU, 1.75GB RAM, scales up to 3 instances
-              const: B1
-            - title: 2 vCPU, 3.5GB RAM, scales up to 3 instances
-              const: B2
-            - title: 4 vCPU, 7GB RAM, scales up to 3 instances
-              const: B3
-            - title: 1 vCPU, 1.75GB RAM, scales up to 10 instances
+            - title: 1 vCPU, 1.75GB RAM
```
Comment (WillBeebe) [inline azure-app-service/massdriver.yaml:146]: @mclacore I think we can do a follow-up PR for these. Either a list or we can do a mini hackathon together and get it into Satellite. 
🤙

File: azure-app-service/massdriver.yaml [hunk 3]
```diff
@@ -204,35 +187,6 @@ params:
           title: Image Tag
           $ref: https://raw.githubusercontent.com/massdriver-cloud/artifact-definitions/main/definitions/types/docker-image-tag.json
           default: latest
-    network:
```
Comment (WillBeebe) [inline azure-app-service/massdriver.yaml:207]: The app template sets up the network automatically. If you'd like a more complicated template... fork the repo. :)

**Review by likelymichael — APPROVED** [review]
Nice work

**Pull request merged**